jupyterlab_git/git.py

```diff
@@ -339,7 +343,7 @@ def __init__(self, contents_manager):
                     note[count] = words[i]
                     count += 1
             for num in range(1, int(length / 2)):
-                line_info = line_array[num].split(maxsplit=2)
+                line_info = line_array[num].split('\t', maxsplit=2)
```

Comment:
why is the `/t` now necessary?

Reply:
Originally by accident, but on further investigation the `-z` flag splits renaming of files onto multiple lines, so now if there are spaces in a name it could get split weirdly. i.e.:
without `-z`:  
`0    0    file1 => renamed file with spaces`
with `-z` that becomes:  
```
0    0
file1
renamed file with spaces```
so the `maxsplit=2` no longer protects from splitting filenames on spaces. Though this needs further modification to fix how it handles when files are renamed. I added a comment to the main thread here asking how to handle renaming of files in the history tab.